help/C/wordcompletion.page

```diff
@@ -0,0 +1,55 @@
+<page xmlns="http://projectmallard.org/1.0/"
+      xmlns:its="http://www.w3.org/2005/11/its"
+      type="topic" style="task"
+      id="plugin-word-completion">
+
+  <info>
+    <link type="guide" xref="pluma-plugin-guide#pluma-additional-plugins"/>
```

Comment:
Use `index` instead of `pluma-plugin-guide` to display the plugin help in the additional plugins section:
```diff
diff --git a/help/C/wordcompletion.page b/help/C/wordcompletion.page
index bd0cd89..87c1753 100644
--- a/help/C/wordcompletion.page
+++ b/help/C/wordcompletion.page
@@ -4,7 +4,7 @@
       id="plugin-word-completion">
 
   <info>
-    <link type="guide" xref="pluma-plugin-guide#pluma-additional-plugins"/>
+    <link type="guide" xref="index#pluma-additional-plugins"/>
 
     <credit type="author">
       <name>Jim Campbell</name>
```
test
```
$ yelp help:pluma-plugins
```